processor/src/main/java/com/blarley/rxretrobusseed/annotationprocessor/processor/RxRetroBusAnnotationProcessor.java

```diff
@@ -56,13 +58,20 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             builder.append("\tprivate " + baseType + " client;\n");
             builder.append("\tprivate RxRetroBus bus;\n\n");
 
-            // Constructor - builds Retrofit client
-            builder.append("\tpublic " + generatedClassName + "(Retrofit.Builder retrofitBuilder, RxRetroBus bus) { \n" +
-                            "\t\tthis.client = retrofitBuilder.baseUrl(\"" + baseUrl + "\")\n" +
-                            "\t\t\t\t.build()\n" +
-                            "\t\t\t\t.create(" + baseType + ".class);\n" +
-                            "\t\tthis.bus = bus;\n" +
-                            "\t}\n\n");
+            if (generateEvents.retrofit()) {
+                // Constructor - builds Retrofit client
+                builder.append("\tpublic " + generatedClassName + "(Retrofit.Builder retrofitBuilder, RxRetroBus bus) { \n" +
```

Comment:
This might not be necessary, I think we can just pass in null to the builder portion and key off that